src/strategy/immediate.rs

```diff
@@ -19,20 +16,6 @@ pub(crate) struct StrategyImmediate {
 }
```

Comment:
Do we still need the `check` and `finalize` booleans in the struct?

Reply:
Probably not, unless somebody comes and ask back for this later on in the future. I was mostly concerned about not exposing the configuration knobs. Do you mind if I leave these for the time being, and revisit after a few releases?

Reply:
Yup, WFM!

Reply:
Tracked a reminder ticket at https://github.com/coreos/zincati/issues/35.